MQ2PluginManager.cpp

```diff
@@ -161,20 +124,27 @@ void DrawPluginManager_MQSettingsPanel()
 	}
 }
 
-PLUGIN_API void OnCleanUI()
+void DrawPluginManager_MQSettingsPanel()
 {
-	DestroyPluginWindow();
+	DrawGUI();
 }
 
-PLUGIN_API void OnReloadUI()
+PLUGIN_API void OnUpdateImGui()
 {
-	CreatePluginWindow();
-}
+	if (!s_showWindow)
+		return;
 
-PLUGIN_API void SetGameState(int GameState)
```

Comment:
Changed my mind about this a little bit. Perhaps looking for additional feedback on this...

I don't know if we want to be immediately showing this at login by default, but would want it to appear in game by default like it did before. So, maybe make it toggleable at any point in the game, but only appear based base on last state when it was in game?